src/arch/svm/eventsClient.ts

```diff
@@ -9,10 +9,13 @@ import web3, {
   Signature,
 } from "@solana/kit";
 import { bs58 } from "../../utils";
-import { EventData, EventName, EventWithData } from "./types";
+import { EventWithData } from "./types";
 import { decodeEvent, isDevnet } from "./utils";
 import { getSlotForBlock } from "./SpokeUtils";
 
+// Blanket type definition so that we don't need to import the `Idl` type from anchor directly.
+type Idl = typeof SvmSpokeIdl;
```

Comment:
Is this `Idl` generic, or specific to the `SvmSpokeIdl` in any way?

fwiw we already rely on anchor for the `Idl` type in `utils.ts`, so I think it's OK to re-export `Idl` from there.

Reply:
Good point https://github.com/across-protocol/sdk/pull/982/commits/5f3d674305951620c0567d116fe63096b6dba173